src/components/CDPCreateScreens/CDPCreateConfirmCDP.js

```diff
@@ -228,6 +230,7 @@ const CDPCreateConfirmCDP = ({ dispatch, cdpParams, selectedIlk, onClose }) => {
         daiToDraw
       );
 
+    setEnableSubmit(false);
```

Comment:
for the case where somebody rejects a transactions, do you think it makes sense to re-enable submit?

Reply:
Yeah, that's a good catch. I'll reenable the button if we get an error with the tx